rare.sql

```diff
@@ -48,4 +48,3 @@ CREATE TABLE `PostTag` (
     FOREIGN KEY(`post_id`) REFERENCES `Posts`(`id`),
```

Comment:
Will taking these out of the sql file cause any problems later? You'll need them if you ever have to create the table again for any reason